internal/telemetry/package_source_darwin.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build darwin
+// +build darwin
+
+package telemetry
+
+import (
+	"os"
+)
+
+func readPackageSource() *string {
+	if b, err := os.ReadFile("/usr/local/etc/atlascli/package_source"); err == nil {
```

Comment:
for mac what's suppose to create this file?

Reply:
homebrew once a PR is accepted and merged, there is a comment in the PR description

Reply:
This is possible, I forgot about it